Dockerfile

```diff
@@ -0,0 +1,63 @@
+
+# /=====================================================================\ #
+# | LaTeXML Dockerfile                                                  | #
+# | A Dockerfile to create a Docker Image with LaTeXML preinstalled.    | #
+# |=====================================================================| #
+# | Thanks to Tom Wiesing <tom.wiesing@gmail.com>                       | #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+
+# We start from alpine linux 3.7
+FROM alpine:3.7
+
+# Install the dependencies
+RUN apk add --no-cache \
+    curl \
+    db-dev \
+    gcc \
+    libc-dev \
+    libgcrypt \
+    libgcrypt-dev \
+    libxml2 \
+    libxml2-dev \
+    libxslt \
+    libxslt-dev \
+    make \
+    perl \
+    perl-dev \
+    perl-utils \
+    wget \
+    zlib \
+    zlib-dev
+
+# Install cpanmnius
+RUN curl -L https://cpanmin.us | perl - App::cpanminus
```

Comment:
same as in the other PR, you could add the `cpanminus` dependency to the `apk` section, instead of installing from curl, then it feels a bit more robust for a deployment setup.